dot/parachain/candidate-validation/candidate_validation_test.go

```diff
@@ -19,12 +19,14 @@ import (
 )
 
 var (
-	povHashMismatch  = PoVHashMismatch
-	paramsTooLarge   = ParamsTooLarge
-	codeHashMismatch = CodeHashMismatch
-	badSignature     = BadSignature
-	invalidOutputs   = InvalidOutputs
-	badParent        = BadParent
+	povHashMismatch     = PoVHashMismatch
+	paramsTooLarge      = ParamsTooLarge
+	codeHashMismatch    = CodeHashMismatch
+	badSignature        = BadSignature
+	invalidOutputs      = InvalidOutputs
+	badParent           = BadParent
+	invalidSessionIndex = InvalidSessionIndex
+	invalidCoreIndex    = InvalidCoreIndex
```

Comment:
is there a reason to not use just the normal value like `PoVHashMismatch`? why do we need to create a new var?

Reply:
in validationResult structure, both `Valid` and `Invalid` are pointer type. thus we want to store the address of a variable to `Invalid` field in this case.

`PoVHashMismatch ` is a constant. In Go, you cannot get the memory address of a const. So we need to create variable and then use the address of this variable.